azure-pipelines.yml

```diff
@@ -284,6 +284,18 @@ stages:
             inputs:
               artifactName: 'Envoy.framework'
               targetPath: 'dist/Envoy.framework'
+      - job: swift_library_tests
+        timeoutInMinutes: 60
+        pool:
+          vmImage: 'macos-10.14'
+        steps:
+          - checkout: self
+            submodules: true
+          - script: ./ci/mac_ci_setup.sh
+            displayName: 'Install dependencies'
+          - script: |
+              ./bazelw test --test_output=all --build_tests_only //library/swift/test/...
+            displayName: 'Run Swift library tests'
```

Comment:
looks like this isn't interpreted correctly because of the multiline string above

Reply:
Seems to have been working for Android, but updated

Reply:
maybe it's just github's rendering then, but i think this is nicer

Reply:
Yep same